tests/playbooks/tests_states.yml

```diff
@@ -47,3 +47,32 @@
             state: down
     - include_tasks: tasks/assert_device_absent.yml
     - include_tasks: tasks/assert_profile_absent.yml
+
+    #test case (downing a non existent profile)
+    #I can set down and remove a non existent profile
+    #This will be rescued until issue 188 is fixed
```

Comment:
```suggestion
    # test case (downing a non existent profile)
    # I can set down and remove a non existent profile
    # This will be rescued until issue #188 is fixed
```